packages/lib/src/components/Dropin/components/PaymentMethod/PaymentMethodBrands/PaymentMethodBrands.tsx

```diff
@@ -30,7 +41,11 @@ const PaymentMethodBrands = ({ brands, excludedUIBrands = [], isPaymentMethodSel
             {visibleBrands.map(brand => (
                 <PaymentMethodIcon key={brand.name} altDescription={getFullBrandName(brand.name)} type={brand.name} src={brand.icon} />
             ))}
-            {leftBrandsAmount !== 0 && <span className="adyen-checkout__payment-method__brand-number">+{leftBrandsAmount}</span>}
+            {showOtherInsteafOfNumber ? (
+                <span className="adyen-checkout__payment-method__brand-number">+ {i18n.get('paymentMethodBrand.other')}</span>
+            ) : (
+                leftBrandsAmount !== 0 && <span className="adyen-checkout__payment-method__brand-number">+{leftBrandsAmount}</span>
+            )}
```

Comment:
Minor thing, but maybe you can add a simple test case to validate this part

Reply:
Yeah, the only problem of this is that we need to validate it in the Dropin, since is not really a PM logic.

Reply:
There is already a `PaymentmethodBrands.test.tsx` - file , perhaps you can just add this there? Just a new test case passing the new props and making sure it renders ok